packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -449,13 +449,19 @@ if (import.meta.hot) {
 					}
 
 					if (hasDevContainers) {
+						viteDevServer.config.logger.warn(
+							"Building container images for local development..."
+						);
```

Comment:
```suggestion
						viteDevServer.config.logger.info(
							"Building container images for local development..."
						);
```

Unless there was a reason you chose `warn`?